pkg/statser/statser_internal.go

```diff
@@ -34,7 +34,7 @@ type InternalStatser struct {
 
 // NewInternalStatser creates a new Statser which sends metrics to the
 // supplied InternalHandler.
-func NewInternalStatser(bufferSize int, tags gostatsd.Tags, namespace, hostname string, handler InternalHandler) Statser {
+func NewInternalStatser(bufferSize int, tags gostatsd.Tags, namespace, hostname string, handler InternalHandler) *InternalStatser {
```

Comment:
I removed the `Run` method from the `Statser` interface (because it is an implementation detail that you need to run this statser) and hence I need to export the type that exports the `Run` method. This is the same as `Dispatcher` interface and `MetricDispatcher` impl.